tests/conftest.py

```diff
@@ -38,38 +41,16 @@
               [1.61296440 + 0.3498192j, 1.23920938 + 0j]])
 
 
-# ==========================================================
-# PennyLane devices
-
-# List of all devices that support analytic expectation value
-# computation. This generally includes statevector/wavefunction simulators.
-analytic_devices = [QulacsDevice]
-
-# List of all devices that do *not* support analytic expectation
-# value computation. This generally includes hardware devices
-# and hardware simulators.
-hw_devices = []
-
-# List of all device shortnames
-shortnames = [d.short_name for d in analytic_devices + hw_devices]
-
-
 # ==========================================================
 # pytest fixtures
 
-@pytest.fixture
-def tol(shots):
-    """Numerical tolerance to be used in tests."""
-    if shots == 0:
-        # analytic expectation values can be computed,
-        # so we can generally use a smaller tolerance
-        return {"atol": 0.01, "rtol": 0}
 
-    # for non-zero shots, there will be additional
-    # noise and stochastic effects; will need to increase
-    # the tolerance
-    return {"atol": 0.05, "rtol": 0.1}
+TOL = 1e-3

```

Comment:
Were there some cases where tolerance of this magnitude was necessary for passing tests? (Not too crucial just curious --- we'd usually have a smaller tolerance value I think in PennyLane core)

Reply:
True, I set it to ``1e-8``